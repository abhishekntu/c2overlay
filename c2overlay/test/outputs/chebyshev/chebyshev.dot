digraph graphname {
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="N2_mul_N2_N2"];
N3 [color="black", label="N3_mul_N3_N3"];
N4 [color="black", label="N4_mul_Imm_16_N4_N4"];
N5 [color="black", label="N5N7_mul_N5_sub_Imm_20_N7_N5"];
N6 [color="black", label="N6N8_mul_N6_add_Imm_5_N8_N6"];
N7 [color="black", label="O0_N9_N7"];
N1 -> N5;
N1 -> N6;
N1 -> N2;
N1 -> N3;
N1 -> N4;
N2 -> N7;
N3 -> N6;
N4 -> N5;
N5 -> N3;
N6 -> N2;
}
