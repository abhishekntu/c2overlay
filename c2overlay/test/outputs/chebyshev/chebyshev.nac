procedure foo(in s32 x,out s32 D_2404)
{
  localvar s32 D_2405;
  localvar s32 D_2406;
  localvar s32 D_2407;
  localvar s32 D_2408;
  localvar s32 D_2409;
  localvar s32 temp;
L0005:
  temp <= mul x,16;
  D_2405 <= mul temp,x;
  D_2406 <= sub D_2405,20;
  D_2407 <= mul D_2406,x;
  D_2408 <= mul D_2407,x;
  D_2409 <= add D_2408,5;
  D_2404 <= mul D_2409,x;
}
