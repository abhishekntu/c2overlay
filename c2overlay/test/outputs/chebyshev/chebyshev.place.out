Netlist file: test.net   Architecture file: test.arch.xml
Array size: 3 x 3 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N1		3	0	0	#0
out:N7		1	0	0	#1
N2_blk		1	1	0	#2
N3_blk		3	1	0	#3
N4_blk		2	2	0	#4
N5_blk		3	2	0	#5
N6_blk		2	1	0	#6
